src/components/navigation-header/components/product-page-content/utils/get-nav-items.ts

```diff
@@ -33,11 +33,12 @@ export function getNavItems(currentProduct: ProductData): NavItem[] {
 			label: 'Tutorials',
 			url: `/${currentProduct.slug}/tutorials`,
 		},
-		{
+	]
+	currentProduct.slug !== 'hcp' &&
```

Comment:
HCP doesnt have an 'install' link or page, there are a few changes that adjusts the auto gen for items to remove this.